app/models/user.rb

```diff
@@ -57,6 +57,8 @@ def name
   end
 
   def percentile_rank
+    return 0 unless record_activity?
```

Comment:
I think it might be preferable to hide the stats widget on `/account`, rather than display all zeros. Seeing zeros there might confuse users, and if they took actions before they opted out of tracking they'll see nonzero, but still incorrect, numbers.